packages/@react-spectrum/table/src/Table.tsx

```diff
@@ -238,7 +238,8 @@ function TableCollectionView({layout, collection, focusedKey, renderView, render
     onVisibleRectChange(rect) {
       bodyRef.current.scrollTop = rect.y;
       setScrollLeft(bodyRef.current, direction, rect.x);
-    }
+    },
+    transitionDuration: collection.body.props.isLoading ? 0 : 500
```

Comment:
Can we disable it only for load more? I think it looked good for initial loading... You should be able to check if the collection has items in it already to determine if it's loading more.